core/api/src/graphql/public/schema.graphql

```diff
@@ -338,6 +338,21 @@ type Currency {
   symbol: String!
 }
 
+type CurrencyConversionEstimate {
+  """Amount in satoshis."""
+  btcSatAmount: SatAmount!
+  currency: Currency!
```

Comment:
should we also return the displayCurrency? that we have as input? for cache purpose for instance

Reply:
It is not necessary. Removed https://github.com/GaloyMoney/galoy/pull/4222/commits/bc722bc1d5c06b0a98c9c32669b084091f774c82